static/app/components/core/checkbox/index.chonk.tsx

```diff
@@ -0,0 +1,48 @@
+import {chonkStyled} from 'sentry/utils/theme/theme.chonk';
+
+export const ChonkNativeHiddenCheckbox = chonkStyled('input')`
+  position: absolute;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  cursor: pointer;
+
+  & + * {
+    color: ${p => p.theme.colors.static.white};
+    border: 1px solid ${p => p.theme.colors.dynamic.surface100};
+
+    svg {
+      stroke: ${p => p.theme.colors.static.white};
+    }
+  }
+
+  &:focus-visible + * {
+    outline: none;
+    box-shadow: 0 0 0 2px ${p => p.theme.colors.dynamic.surface100},
+      0 0 0 4px ${p => p.theme.focusBorder};
```

Comment:
since this is the same one we have in `input`, would it make sense to standardize it for all form components?

actually, it’s slightly different: you’re  using `theme.colors.dynamic.surface100` while I was using `theme.background`. I thought they might map to the same actual value, but it looks like `theme.background` is actually `theme.colors.dynamic.surface500`. I’m a bit confused now what’s correct and how we should do those 🤔 ...